nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Operation.java

```diff
@@ -101,19 +104,20 @@ public String getOperationId() {
     public <T> T execute(String operationId, OperationBody body) {
         Object input = body.getInput();
         if (input instanceof Blob) { // If input is blob or blobs -> use multipart
```

Comment:
Add `Blob blob = (Blob) input;` then use that instead of casting each time. Same for `Blobs blobs = (Blobs) input;` below.